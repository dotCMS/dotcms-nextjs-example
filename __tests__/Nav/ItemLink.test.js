import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';


describe('<RouterLink />', () => {
    it('renders and matches snapshot', () => {
        window.history.pushState({}, 'Test Title', '/test.html?lang=en');
        const props = {
            className: 'testCss',
            pathname: 'testPath',
            state: {
                id: 'testId',
                name: 'testName'
            }
        };
        const wrapper = shallow(<RouterLink {...props} />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});
