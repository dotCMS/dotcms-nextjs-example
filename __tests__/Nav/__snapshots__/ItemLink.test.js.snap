// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RouterLink /> renders and matches snapshot 1`] = `
<Link
  className="testCss"
  to={
    Object {
      "pathname": "testPath",
      "state": Object {
        "id": "testId",
        "name": "testName",
      },
    }
  }
/>
`;
